### DISCUSSION TITLE
1450和反华分子都是在世的活菩萨！

### DISCUSSION THREAD

**Discussion by Ventery** [WeNeedHome/SummaryOfLoanSuspension#1147] [Ideas]
共产党这么坏，那就冷眼旁观其自取灭亡好了！干嘛还要苦口婆心去建议大陆施行民主自由呢？
可见这些1450和这些反华分子的目的绝不是颠覆大陆政府，而是想要救中国人民于水火。
真是当代活菩萨啊！👍👍👍👍👍

**Comment by Ventery** [comment]
各位1450和反华的朋友们，咱们一起来认真讨论一下

**Reply by Humphrey75432** [reply]
他们不是为了拯救中国人民，是把你搞垮了以后再收割你。要当坏比就要当得彻底一些。

**Comment by itsoursduty** [comment]
大家都是想救自己而已

**Comment by Cegazhos** [comment]
如果只是批评政府银行开发商，我还能给他点个赞。可这群杂种偏偏要说买期房的过错

肯定是莱猪肉太好吃了，才导致这群台湾乡巴佬脑子里全是激素。
点名那个傻狗 JayChou55 都暴露成这样了 还搁那装
想来也是跟他老母学的冰清玉洁吧

**Comment by Cegazhos** [comment]
中修，米国，小日本，台湾狗，我都感到厌恶。
但要排优先级，我觉得应该先把台湾傻狗统统干掉

**Comment by Ventery** [comment]
怎么这么久了这群畜生还没上钩😅

**Reply by xijinping0** [reply]
楼主是一定要突开化粪池是吧？😅

[![突开](https://img.youtube.com/vi/B8V4rrKuTkE/0.jpg)](https://youtu.be/B8V4rrKuTkE?t=113)

https://github.com/xijinping0
《[习近平与他的情人们](https://lovers.xijinping.one)》

**Reply by b4o** [reply]
> 楼主是一定要突开化粪池是吧？😅
> 
> [![突开](https://camo.githubusercontent.com/49034fe6cb4ed072e8cca5665df0bfb4a42a26af42294a24088cbc5e9637e941/68747470733a2f2f696d672e796f75747562652e636f6d2f76692f4238563472724b75546b452f302e6a7067)](https://youtu.be/B8V4rrKuTkE?t=113)
> 
> https://github.com/xijinping0 《[习近平与他的情人们](https://lovers.xijinping.one)》

你拜登爷爷拉西了，你去给他舔了吗？

**Comment by frshgrass** [comment]
祝好：祝你早日吃着特供住着高干病房，变成体制人上人。在大清当八旗还是恒爽的。几万万黔首996赚美元，你们天天吃空响拿国企利润分脏！像你们的华姐和坚哥美国有大house！共f党新的封建主义还是进步性。至少什么国企一把手，各种实权部门官员这样的封建领主不需要什么血缘关系。祝好啊！早点人上人！红色贵族永流传！❤️
